<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>serviceContent</class>
 <widget class="ServiceContent" name="serviceContent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>568</width>
    <height>439</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Service</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="buttonCopy">
     <property name="text">
      <string>Kopie (F1)</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Schl√ºssel-Nummer</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="editKey"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="1">
      <widget class="QLineEdit" name="editMainDescription"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Haupt-Bezeichnung:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Extra-Information:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="PageTextEdit" name="editDescr">
       <property name="tabChangesFocus">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Einheit:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="editUnit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="4">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Gesamt</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Material</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Material EKP</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Hilfsmaterial</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Leistung</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>(min)</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>(EUR)</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="editServicePeriod">
       <property name="text">
        <string>0,0</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLineEdit" name="editMatPrice">
       <property name="text">
        <string>0,0</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLineEdit" name="editHelperMatPrice">
       <property name="text">
        <string>0,0</string>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QLabel" name="labelTotal">
       <property name="text">
        <string>0.00</string>
       </property>
      </widget>
     </item>
     <item row="2" column="5">
      <widget class="QLineEdit" name="editMatEkp">
       <property name="text">
        <string>0,0</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="labelServicePrice">
       <property name="text">
        <string>0.0</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PageTextEdit</class>
   <extends>QTextEdit</extends>
   <header>functionality/utils.h</header>
  </customwidget>
  <customwidget>
   <class>ServiceContent</class>
   <extends>QWidget</extends>
   <header>pages/service_page.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>editKey</tabstop>
  <tabstop>editMainDescription</tabstop>
  <tabstop>editDescr</tabstop>
  <tabstop>editUnit</tabstop>
  <tabstop>editServicePeriod</tabstop>
  <tabstop>editMatPrice</tabstop>
  <tabstop>editHelperMatPrice</tabstop>
  <tabstop>editMatEkp</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <slots>
  <slot>CopyData(QString)</slot>
 </slots>
</ui>
